__tests__/integration/extract/managed-by-pnpm/local-packages/qux/package.json

```diff
@@ -1,5 +1,7 @@
 {
   "name": "qux",
   "version": "4.5.6",
-  "license": "MIT"
+  "license": "MIT",
+  "author": "Eve Engineer <eve@example.org>",
+  "repository": "https://gitlab.com/example/qux"
```

Comment:
Could you add a test case for the pattern where author and repository do not exist in package.json?